quickstarts/Safety.ipynb

```diff
@@ -61,7 +61,7 @@
       "source": [
```

Comment:
<blockquote><code>* HARM_CATEGORY_DANGEROUS</code></blockquote><blockquote> * You can refer to safety settings ...</blockquote><p> </p><p>This doesn't looks like it's meant to be in the same list.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>fixed.</p>